features/steps/partcad-cli/commands/add/assembly.py

```
@@ -15,6 +15,14 @@ def step_impl(context, filename):
         logging.debug(f"Creating file: {filename}")
         file.write(content)
 
+@given('a user configuration file named "{filename}" with content')
+def step_impl(context, filename):
```

Ruff (0.8.2)

Missing return type annotation for public function `step_impl`

Add return type annotation: `None`

(ANN201)